src/templates/getHandler.ts

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import type { NextConfig } from '../helpers/config'
+
+const { promises } = require('fs')
+const { Server } = require('http')
+const path = require('path')
+// eslint-disable-next-line node/prefer-global/url, node/prefer-global/url-search-params
+const { URLSearchParams, URL } = require('url')
+
+const { Bridge } = require('@vercel/node/dist/bridge')
+
+const { augmentFsModule, getMaxAge, getMultiValueHeaders, getNextServer } = require('./handlerUtils')
+
+const makeHandler =
+  () =>
```

Comment:
[sand] You can add a return type here, but you'll probably have to function type since it's a long one